apps/cms/templates/meinberlin_cms/home_page.html

```diff
@@ -1,20 +1,49 @@
 {% extends "base.html" %}
-{% load wagtailcore_tags wagtailuserbar %}
+{% load wagtailcore_tags wagtailuserbar wagtailimages_tags rules %}
 
-{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}
+{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}
+    {{ self.title }}{% endif %}{% endblock %}
 
 {% block header %}
     {% if self.search_description %}
-    <meta name="description" content="{{self.search_description}}" />
+        <meta name="description" content="{{ self.search_description }}"/>
     {% endif %}
 {% endblock %}
 
 {% block content %}
+    {% image self.header_image fill-1920x560-c100 as header_image %}
+
+    <div class="hero-unit" style="background-image: url({{ header_image.url }})">
+        <div class="l-wrapper hero-unit__wrapper">
+            <h1 class="hero-unit__title">{{ self.title }}</h1>
+            <h2 class="hero-unit__subtitle">{{ self.subtitle }}</h2>
+        </div>
+    </div>
+
     <div class="l-wrapper">
-        <h1>{{ self.title }}</h1>
+        <h3>Aktuelle Beteiligungsverfahren</h3>
+        <div class="l-tile-wrapper">
+            {% for project in self.featured_projects %}
 
-        {{ self.body|richtext }}
+                {% has_perm 'a4projects.view_project' request.user project as show %}
+                {% if show %}
+                    {% include "a4projects/includes/project_list_tile.html" with project=project %}
+                {% endif %}
+
+            {% endfor %}
+        </div>
     </div>
```

Comment:
You mean a wagtail block? Why would we want to do that? Do you mean for the editorial seöection of featured projects? I didn't do it yet because it was in another user story, but I talked with @MagdaN about it, it can be done. But i'd do that in a separate PR.